tests/test_cli_units.py

```diff
@@ -55,6 +55,19 @@ def test_is_composite_member_with_renderer(self):
         res = NetplanApply.is_composite_member([{'renderer': 'networkd', 'br0': {'interfaces': ['eth0']}}], 'eth0')
         self.assertTrue(res)
 
+    @patch('subprocess.run')
+    def test_get_alt_names(self, mock):
+        stdout_mock = mock.Mock()
+        stdout_mock.stdout = "3: ens4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 8958 qdisc fq_codel state "\
+            "UP mode DEFAULT group default qlen 1000\n"\
+            "    link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff\n"\
+            "    altname enp0s4\n"\
+            "    altname enp0s41".encode('utf-8')
```

Comment:
Sorry for the nitpick, but can we please use single quotes here, as we do in the rest of the file?

Reply:
Ok, will fix that.